src/components/Class/ClassesList/styles.scss

```diff
@@ -0,0 +1,110 @@
+
+  
+
+.createClassButton{
+	background: #83B344;
+	margin-left: 17%;
+	width: 10%;
+	height: 40px;
+	border-radius: 15px;
+	color: white;
+	font-size: 17px;
+	font-weight:'bold';
+	border: none ;
+}
+
+
+
+//  style={{marginLeft:'17%',backgroundColor:'#83B344',width:'10%',height:'40px',borderRadius:'15px',color:'white',fontSize:'17px',fontWeight:'Bold',border:'none'}}
```

Comment:
If we don't need it. We should remove the commented code.

Reply:
Done